test/core/if.wast

```diff
@@ -768,8 +768,6 @@
   "type mismatch"
 )
 
-;; TODO: Compare above "*operand-missing*" tests to others, identify and remove duplicates.
```

Comment:
Can we rename the above to `$type-condition-empty-*`, to closer match the naming scheme of the other tests?

Also, `in-if` could be `in-then` for symmetry.

Reply:
Yeh, good suggestion, I will rename them soon.